Source/msg.cpp

```diff
@@ -1292,13 +1294,15 @@ void DeltaImportJunk(BYTE *src)
 		}
 	}
 
+	mq = sgJunk.quests;
 	for (i = 0; i < MAXMULTIQUESTS; i++) {
 		if (questlist[i]._qflags & 1) {
-			memcpy(&sgJunk.quests[i], src, sizeof(MultiQuests));
+			memcpy(mq, src, sizeof(MultiQuests));

```

Comment:
Is `*mq = src` still binexact here? :)

Reply:
Not likely given you're trying to assign a BYTE* to a MultiQuest struct.